src/services/agile/baseAgileService.test.ts

```diff
@@ -1,19 +1,43 @@
-import { BacklogItem, BacklogItemType, AgileService, Sprint, Project } from "../../models";
+import { AgileService, BacklogItem, BacklogItemType, Project, Sprint } from "../../models";
 import { BaseAgileService } from "./baseAgileService";
-import { defaultBacklogItems, emptyBacklogItems } from "../../samples";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+interface MockAgileServiceFunctions {
+  createProject: any;
+  getSprint: any;
+  deleteSprint: any;
+  getBacklogItems: any;
+  createProviderBacklogItems: any;
+  deleteBacklogItems: any;
+  createProviderSprints: any;
+}
```

Comment:
Can we do `any`thing different with the type system here (badum tiss)

Reply:
Done